app/src/components/Navbar.js

```diff
@@ -0,0 +1,144 @@
+import React, { Fragment } from 'react'
+import { Disclosure, Menu, Transition } from '@headlessui/react'
+import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline'
+import { PlusIcon } from '@heroicons/react/20/solid'
+
+const navigation = [
+  { name: 'Dashboard', href: '#'},
+  { name: 'Skills', href: '#'},
+  { name: 'Courses', href: '#'},
+  { name: 'Job Roles', href: '/roles'},
+]
+const userNavigation = [
+  { name: 'Your Learning Journeys', href: '#' },
+  { name: 'Settings', href: '#' },
+  { name: 'Sign out', href: '#' },
+]
+function classNames(...classes) {
+  return classes.filter(Boolean).join(' ')
+}
+
+function isCurrent(href) {
+  let current_url = document.location.toString().split("/")
+  let page = "/" + current_url[current_url.length-1];
+  if (page == href) {
+    return true;
+  }
+  return false;
+}
+
+export default function Navbar() {
+  return (
+    <Disclosure as="nav" className="bg-cyan-800">
+      {({ open }) => (
+        <>
+          <div className="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8">
+            <div className="flex h-16 justify-between">
```

Comment:
Looks like the div here is very nested. Do we need so many levels? Can we (1) simplify the levels , (2) refactor the divs, or (3) pull out into a function and use a loop instead.

Reply:
dont quite understand what you mean by (3). Removed a level in commit e1a92a9